### ISSUE TITLE
Need documentation on public MSI properties, to automate installation

### ISSUE THREAD

**Issue by RobSeder** [docker/toolbox#536]
On Windows: since the main deliverable is an .exe, versus an .msi, the public properties can't be read, which makes it difficult/impossible to easily automate the installation. I looked in the source, but don't see any files related to the Windows installer.

Please provide documentation on what command-line properties can be passed to the installer .exe, to pre-fill properties, so that the end-user can silently install it in one command. Thanks!

**Comment by bgrissin** [comment]
have you seen this latest release  [](https://docs.docker.com/docker-for-windows/)

**Comment by RobSeder** [comment]
@bgrissin Yes, and thanks for the reply. When I try to install that, I get an error of:

> This version of Docker requires Windows 10 Pro, Enterprise, or Education edition with a minimum build number of 10586. Please use Docker Toolbox.

I'm working on Windows Server 2012 R2 Standard (build 9600). Thanks!

**Comment by galeksandrp** [comment]
Please note that installer uses Inno Setup. Command line is [here](http://www.jrsoftware.org/ishelp/index.php?topic=setupcmdline). For future guessing of unattended installation switches you can use [chocolatey database](https://chocolatey.org/packages/docker-toolbox) or [USSF](https://chocolatey.org/packages/ussf).
